sigeca_data_import_microservice/app/application/email_notification.py

```diff
@@ -0,0 +1,60 @@
+from typing import List
+
+from app.config import Config
+from app.infrastructure.smtp_client import SMTPClient
+
+
+def _emails_formatter(facilities_response):
+    template = """Dear Administrator,
+
+    The recent synchronization process has finished`    .
+
+    Please review the resources below and make the necessary manual adjustments.
+
+    Resources:
+    {}
+    Please manually update the resources listed above and verify the changes.
+    """
```

Comment:
This template will include the 4 space indentation. Please move the templates of the function to fix that